[2025-03-09T14:46:27.978+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-09T14:46:27.998+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_fire_incidents_dag.testing_data manual__2025-03-09T14:46:25.543187+00:00 [queued]>
[2025-03-09T14:46:28.007+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_fire_incidents_dag.testing_data manual__2025-03-09T14:46:25.543187+00:00 [queued]>
[2025-03-09T14:46:28.008+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-09T14:46:28.029+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): testing_data> on 2025-03-09 14:46:25.543187+00:00
[2025-03-09T14:46:28.042+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=16890) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-09T14:46:28.045+0000] {standard_task_runner.py:72} INFO - Started process 16892 to run task
[2025-03-09T14:46:28.045+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_fire_incidents_dag', 'testing_data', 'manual__2025-03-09T14:46:25.543187+00:00', '--job-id', '495', '--raw', '--subdir', 'DAGS_FOLDER/fire_incidents_dag.py', '--cfg-path', '/tmp/tmp9qp8mabm']
[2025-03-09T14:46:28.046+0000] {standard_task_runner.py:105} INFO - Job 495: Subtask testing_data
[2025-03-09T14:46:28.095+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_fire_incidents_dag.testing_data manual__2025-03-09T14:46:25.543187+00:00 [running]> on host fad361e322ea
[2025-03-09T14:46:28.181+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@***.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_fire_incidents_dag' AIRFLOW_CTX_TASK_ID='testing_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T14:46:25.543187+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T14:46:25.543187+00:00'
[2025-03-09T14:46:28.183+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-09T14:46:28.183+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-09T14:46:28.183+0000] {logging_mixin.py:190} INFO - Current task name:testing_data state:running start_date:2025-03-09 14:46:27.998923+00:00
[2025-03-09T14:46:28.183+0000] {logging_mixin.py:190} INFO - Dag name:etl_fire_incidents_dag and current dag run status:running
[2025-03-09T14:46:28.183+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-09T14:46:28.196+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-09T14:46:28.203+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-09T14:46:28.203+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_fire_incidents_dag, task_id=testing_data, run_id=manual__2025-03-09T14:46:25.543187+00:00, execution_date=20250309T144625, start_date=20250309T144627, end_date=20250309T144628
[2025-03-09T14:46:28.221+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-09T14:46:28.221+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-09T14:46:28.222+0000] {logging_mixin.py:190} INFO - Dag name:etl_fire_incidents_dag queued_at:2025-03-09 14:46:25.602167+00:00
[2025-03-09T14:46:28.222+0000] {logging_mixin.py:190} INFO - Task hostname:fad361e322ea operator:PythonOperator
[2025-03-09T14:46:28.259+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-09T14:46:28.277+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-09T14:46:28.279+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
